### PR TITLE
Wrap up initial layout

### PR DESCRIPTION
**Pull request by programarina**
This resolves:
https://trello.com/c/c3lFiFS0
https://trello.com/c/36lDBPw3
https://trello.com/c/iXHF8svM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.DS_STORE
+**/.DS_Store
```
Comment (programarina) [inline .gitignore]: I think this should work. :)

File: source/js/views/main/MainPage.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    render() {
+        return (<div className='mainPage'>
+
+            <header>
+                <h3>Hi, name</h3>
+                <button>Log out</button>
+            </header>
+            <aside>
+                <ul>
+                    <li>January</li>
```
Comment (DjordjeWork) [inline source/js/views/main/MainPage.js]: The cleaner way is to make an array of months, and use `map()` through it rendering each of them.

File: source/css/base/calendar.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+aside{
```
Comment (DjordjeWork) [inline source/css/base/calendar.css]: Don't use HTML elements as main selectors, use classnames instead.

File: source/css/base/calendar.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+aside ul li{
+    padding: 20px;
+    list-style: none;
+    font-weight: 100;
+    color: color(var(--accent-color-disabled));
+    box-shadow: 1px 2px 1px color(var(--light-grey-color));
+    text-align: center;
+}
+@media screen and (max-width: 680px){
+aside{display: none;}
```
Comment (DjordjeWork) [inline source/css/base/calendar.css]: Fix indentation

File: source/css/base/calendar.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+}
```
Comment (DjordjeWork) [inline source/css/base/calendar.css]: Add blank line

File: source/css/base/create.css [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    min-height: 400px;
+}
+
+.createNote button{
+    color: color(var(--accent-color));
+    background-color: color(var(--main-dark-color));
+    padding: 10px;
+    width: 100px;
+    border: none;
+    /* box-shadow: 2px 4px 2px color(var(--main-dark-color)); */
```
Comment (DjordjeWork) [inline source/css/base/create.css]: Delete code that we are not using

**Pull request merged**